src/plugins/data/public/ui/query_string_input/fetch_index_patterns.ts

```diff
@@ -48,5 +48,5 @@ export async function fetchIndexPatterns(
       ? exactMatches
       : [...exactMatches, await savedObjectsClient.get('index-pattern', defaultIndex)];
 
-  return allMatches.map(getFromSavedObject);
+  return allMatches.map(indexPatterns.getFromSavedObject);
```

Comment:
Related to

> Why do we export `getFromSavedObject` on the `indexPatterns` object, instead of directly from the module scope?
> 
> ```js
> export { getFromSavedObject };
> ```

Reply:
indexPatterns is now a namespace for IndexPattern related helpers.
Similar to `esFilters`.
(Open to better naming!)

Reply:
Not sure what you mean by "namespace". FWIW,`esFilters` is in fact an ESM namespace import:

![image](https://user-images.githubusercontent.com/9773803/70045709-d33d3a80-15c4-11ea-9eff-0f1188dc9b0d.png)

but `indexPatterns` is just an exported object:

![image](https://user-images.githubusercontent.com/9773803/70045841-1bf4f380-15c5-11ea-953f-81842cded542.png)

Maybe we should make `indexPatterns` a namespace import as well, at least for consistency. Not sure if there are any code tree shaking benefits.